src/coreclr/gc/gcpriv.h

```diff
@@ -1904,6 +1907,8 @@ class gc_heap
 #ifdef USE_REGIONS
     PER_HEAP_ISOLATED_METHOD bool on_used_changed (uint8_t* left);
 
+    PER_HEAP_ISOLATED_METHOD bool get_card_table_commit_layout (uint8_t* from, uint8_t* to, uint8_t* commit_begins[total_bookkeeping_elements], size_t commit_sizes[total_bookkeeping_elements], size_t new_sizes[total_bookkeeping_elements]);
```

Comment:
it would actually be useful to add a comment here to say how new_sizes is related to commit_sizes.